### PR TITLE
RELEASE: Update NEWS 1.16 RC5

### PR DESCRIPTION
**Pull request by shasson5**
## What
Update NEWS 1.16 RC5

## Why ?
To progress release process

### CODE REVIEW

File: NEWS [hunk 1]
```diff
@@ -108,6 +109,7 @@
 * Fixed compilation flags to support ROCm 6.0
 * Fixed values of D2H_THRESH and latencey params
 * Fixed Cuda memory support for iov datatype
+* Increased max number of agents in ROCM
```
Comment (gleon99) [inline NEWS:112]: `ROCm`

File: NEWS [hunk 2]
```diff
@@ -95,6 +95,7 @@
 * Fixed potential deadlock with cuda_copy and RTR protocol
 * Fixed tag_recv return value on immediate completion  
 * Fixed memory corruption by proper memh handling in tag offload rendezvous
+* Fixed default huge pages allocation
```
Comment (gleon99) [inline NEWS:98]: `Don't allocate huge pages by default`

Comment (yosefe) [reply NEWS:98]: Changed default allocator to not use reserved huge pages

**Review by yosefe — COMMENTED** [review]
seems few fixes are missing for example #9653, #9658

**Review by yosefe — COMMENTED** [review]
[UCS/DATASTRUCT: Fixed FP division by zero.](https://github.com/openucx/ucx/commit/f8fa6ffb67234140f6fc7e9b6323ea7e860e2a11)
[CONFIG/M4: Disable MAD support to prevent libibmad/libumad dependency leak](https://github.com/openucx/ucx/commit/b2c1c7a35b0e517b1dc8ee9692d5b87d18174992)
[BUILD/DEBIAN: Remove libnvidia-compute version dependency](https://github.com/openucx/ucx/commit/63c39199db7c0511630dc21249ca7b9819f0f4ec)
[UCP/RCACHE: Fix ibv_reg_mr error by registering memory in rcache callback](https://github.com/openucx/ucx/commit/0511ccb299c91e30c3d50fd9b3fc2d7e3a3905ad)

File: NEWS [hunk 3]
```diff
@@ -130,7 +131,9 @@
 * Enhanced multi-thread test output
 #### Build
 * Fixed JUCX package publishing, so it will include support for ARM
-* Fixed ROCM building and testing
+* Fixed ROCm building and testing
+#### AZP
+* Fixed CUDA CI issues
```
[the remark this answers is not in the document]
Comment (gleon99) [reply NEWS:136]: @yosefe 
https://github.com/openucx/ucx/pull/9682#pullrequestreview-1874739048
https://github.com/openucx/ucx/pull/9658

[the remark this answers is not in the document]
Comment (yosefe) [reply NEWS:136]: sorry, it was copy paste issue https://github.com/openucx/ucx/pull/9658 is not relevant

[the remark this answers is not in the document]
Comment (yosefe) [reply NEWS:136]: remove mentioning the fix of CI issues from the news

File: NEWS [hunk 4]
```diff
@@ -120,7 +121,8 @@
 * Fixed a deadlock when forked debugger is attached during an error in rcache operation
 * Fixed crash due to passing null pointer to log function
 * Fixed crash due to incorrect hashing method
-* Fixed crash in configuration parser cleanup by moving it after profiler cleanup 
+* Fixed crash in configuration parser cleanup by moving it after profiler cleanup
+* Fixed floating point division by zero
```
Comment (yosefe) [inline NEWS:125]: Fixed floating point division by zero during protocols initialization

File: NEWS [hunk 5]
```diff
@@ -128,7 +133,9 @@
 * Enhanced multi-thread test output
 #### Build
 * Fixed JUCX package publishing, so it will include support for ARM
-* Fixed ROCM building and testing
+* Fixed ROCm building and testing
+* Removed libnvidia-compute version dependency
+* Removed libibmad/libumad from default external libraries list
```
Comment (yosefe) [inline NEWS:138]: Removed libibmad/libumad from default build configuration to avoid runtime dependency

**Review by yosefe — APPROVED** [review]
pls squash

**Comment by shasson5** [comment]
@yosefe can merge?

**Pull request merged**